src/ert/logging/job_runner_log.conf

```diff
@@ -0,0 +1,20 @@
+version: 1
+disable_existing_loggers: false
```

Comment:
The failing test in libres tried to assert the content of `caplog`. When running the test individually everything passed, however, if the `test_job_dispatch.py` was run in advance, later tests failed. This was observed even with a trivial test function that shouldn't do anything at all:

`pytest tests/libres_tests/job_runner/test_job_dispatch.py::test_no_jobs_json_file tests/libres_tests/res/enkf/test_load_forward_model.py`

The problem with the failing test in libres was that when we are setting the logging config, the existing loggers are disabled. This is not a problem when running ert as it is expected to (in its own process), but in the tests we run the function from `job_dispatch.py` individually, and then disabling the already configured loggers..

https://docs.python.org/3/library/logging.config.html#logging.config.fileConfig

> disable_existing_loggers – If specified as False, loggers which exist when this call is made are left enabled. 
> The default is True because this enables old behaviour in a backward-compatible way.

Reply:
No longer relevant after refactor. Interesting find though..